src/Modal.js

```diff
@@ -41,7 +41,7 @@ module.exports = {
         var closeDialog = function() {
             ReactDOM.unmountComponentAtNode(self.getOrCreateContainer());
 
-            if (props && props.onFinished) props.onFinished.apply(null, arguments);
+            if (props && props.onFinished) props.onFinished.call(null, false);
```

Comment:
This doesn't look right - i assume someone passed in closeDialog's  `arguments` to the onFinished prop for a reason...  What is the rationale for switching from apply() to call()?